aten/src/ATen/native/UpSample.h

```diff
@@ -3,6 +3,43 @@
 #include <ATen/ATen.h>
 #include <ATen/TensorUtils.h>
 
+
+/**
+ * Note [compute_scales_value]
+ * Note [area_pixel_compute_scale]
+ * ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ * Interpolate with scale_factors can have different behaviors
+ * depending on the value of use_scale_factor:
+ *
+ * - With use_scale_factor = False (current default behavior):
```

Comment:
this comment looks out of date.

Reply:
Do you mean `False` should be `None`? I don't have enough context here, but by looking at the code I think other than that the description is accurate. I would include this change if needed in the separate PR as well.